.agent-os/specs/2025-08-22-eslint-prettier-setup/sub-specs/technical-spec.md

```
@@ -18,30 +18,33 @@ This is the technical specification for the spec detailed in @.agent-os/specs/20
 ## Approach Options
 
 **Option A: Copy @orchestr8 configuration exactly**
+
 - Pros: Proven configuration, consistent with existing codebase, includes perfectionist
 - Cons: May include rules not needed for this smaller project
 
 **Option B: Minimal ESLint setup with Prettier** (Selected)
+
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)